### ISSUE TITLE
Number of errors on the output

### ISSUE THREAD

**Issue by BryanMoslo** [wawandco/milo#54]
It would be great if Milo displays the number of Found Errors at the end of the result 👍 Thanks guys!!

**Comment by paganotoni** [comment]
We currently show the following (as an example) when faults are found:
```
Running all reviewers, see more details in: https://github.com/wawandco/milo#configuration.

assessment/index.html:80:1: alt-required: Alt attribute required for images, areas and input[type=image] (0012:attribute/alt-required)
...

faults found
```

My proposal is to change that last message to be something like:

```
Running all reviewers, see more details in: https://github.com/wawandco/milo#configuration.

assessment/index.html:80:1: alt-required: Alt attribute required for images, areas and input[type=image] (0012:attribute/alt-required)
...

23 faults found
```

**Comment by paganotoni** [comment]
Solved by #60
